mars-cli/mars_cli.py

```diff
@@ -81,6 +81,23 @@
                 fallback="https://wwwdev.ebi.ac.uk/ena/submit/webin/auth/token",
             ),
         },
+        "METABOLIGHTS": {
+            "SERVICE": config.get(
+                "metabolights",
+                "development-url",
+                fallback="https://www-test.ebi.ac.uk/metabolights/mars/ws3/submissions/",
+            ),
+            "SUBMISSION": config.get(
+                "metabolights",
+                "development-submission-url",
+                fallback="https://www-test.ebi.ac.uk/metabolights/mars/ws3/submissions/",
+            ),
+            "TOKEN": config.get(
+                "metabolights",
+                "development-token-url",
+                fallback="https://www-test.ebi.ac.uk/metabolights/mars/ws3/auth/token",
```

Comment:
To be complete, I think these need to be added to [generate_config.py](https://github.com/elixir-europe/MARS/blob/main/mars-cli/generate_config.py)